CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+## [3.1.8] - 2023.09.22
+### Added
+- Generic extension point for Extension Ecosystem
```

Comment:
Can you extend this pls? Its not enough, really :-)

Reply:
Yes this should be really long..

Reply:
extended